index.html

```diff
@@ -433,8 +505,19 @@
       var songList = [];
       var selectedSongIndex;
 
-      //creating a function songDetail to check if it exists youtubeLink variable inside of div songDetail, if it does, so it returns id youtubeLink variable, it it doesnt, it reurns string empty "
-      var songDetail = (youtubeLink) => (`<div class="song-detail">${youtubeLink ? youtubeLink.id : ""} </div>`);
+      //creating a function songDetail to check if it exists youtubeLink variable inside of div songDetail, if it does, so it returns id youtubeLink variable, it it doesnt, it returns empty string"
+      /*var songDetail = function (youtubeLink) {
```

Comment:
Let's remove this unused code

Reply:
I would rather to keep it because if I need it in the future for other projects